enterprise/cmd/frontend/internal/repos/webhooks/github_webhook_handler.go

```diff
@@ -30,15 +33,15 @@ func NewGitHubWebhookHandler() *GitHubWebhookHandler {
 	}
 }
 
-func (g *GitHubWebhookHandler) handleGitHubWebhook(ctx context.Context, _ database.DB, _ extsvc.CodeHostBaseURL, payload any) error {
+func (g *GitHubWebhookHandler) handle(ctx context.Context, payload any) error {
```

Comment:
This confused me in other parts of the code too: `handle` is awfully generic but inside it makes assumptions about the event type it's called for. So I think it's easier to understand when encoded in name

```suggestion
func (g *GitHubWebhookHandler) handlePush(ctx context.Context, payload any) error {
```